plugins/obs-libfdk/obs-libfdk.c

```diff
@@ -133,8 +133,8 @@ static void *libfdk_create(obs_data_t *settings, obs_encoder_t *encoder)
 		break;
 
 		/* lib_fdk-aac > 1.3 required for 7.1 surround;
-         * uncomment if available on linux build
-         */
+		 * uncomment if available on linux build
+		 */
```

Comment:
If this is about `case 8`, then the first line should probably be de-indented, and then the following lines made to match that.

Perhaps this should be placed inside the `ifndef` so that it is not separated from the section it's commenting about.

Reply:
I will make another PR #6393 to remove the outdated code. This change is removed.